Unknown Error: list index out of range

### Version Number

1.20.0

### What branch are you on?

master

### Describe the Bug

```
[2024-03-13 19:42:58,500] [overlays.py:396]           [INFO]     |================ Gathering Items for TV_Top_TextCenter_Returning_24/03/2024 Overlay ================|
[2024-03-13 19:42:58,500] [builder.py:2878]           [INFO]     |                                                                                                    |
[2024-03-13 19:42:58,501] [builder.py:2880]           [INFO]     | Collection Filter last_episode_aired.before: 2024-01-28                                            |
[2024-03-13 19:42:58,501] [overlays.py:404]           [DEBUG]    |                                                                                                    |
[2024-03-13 19:42:58,501] [overlays.py:405]           [DEBUG]    | Builder: tmdb_discover: {'limit': 1000, 'air_date.gte': '03/24/2024', 'air_date.lte': '03/24/2024', 'with_status': 0} |
[2024-03-13 19:42:58,501] [overlays.py:406]           [INFO]     |                                                                                                    |
[2024-03-13 19:42:58,615] [tmdb.py:357]               [INFO]     | Processing TMDb Discover: 149 Shows                                                                |
[2024-03-13 19:42:58,615] [tmdb.py:359]               [INFO]     |            air_date.gte: 2024-03-24                                                                |
[2024-03-13 19:42:58,616] [tmdb.py:359]               [INFO]     |            air_date.lte: 2024-03-24                                                                |
[2024-03-13 19:42:58,616] [tmdb.py:359]               [INFO]     |            with_status: 0                                                                          |
[2024-03-13 19:42:58,616] [builder.py:2115]           [DEBUG]    |                                                                                                    |
[2024-03-13 19:42:58,616] [builder.py:2116]           [DEBUG]    | 149 IDs Found                                                                                      |
[2024-03-13 19:42:58,616] [builder.py:2118]           [DEBUG]    |                                                                                                    |
[2024-03-13 19:42:58,716] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 63935                                               |
[2024-03-13 19:42:58,734] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 6538                                                |
[2024-03-13 19:42:58,746] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 237952                                              |
[2024-03-13 19:42:58,756] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 80079                                               |
[2024-03-13 19:42:58,762] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 246244                                              |
[2024-03-13 19:42:58,774] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 246878                                              |
[2024-03-13 19:42:58,779] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 246878                                              |
[2024-03-13 19:42:58,785] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 246425                                              |
[2024-03-13 19:42:58,792] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 133193                                              |
[2024-03-13 19:42:58,797] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 245981                                              |
[2024-03-13 19:42:58,801] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 244675                                              |
[2024-03-13 19:42:58,807] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 238745                                              |
[2024-03-13 19:42:58,814] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 245986                                              |
[2024-03-13 19:42:58,820] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 196267                                              |
[2024-03-13 19:42:58,825] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 245980                                              |
[2024-03-13 19:42:58,830] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 217335                                              |
[2024-03-13 19:42:58,836] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 246843                                              |
[2024-03-13 19:42:58,842] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 158733                                              |
[2024-03-13 19:42:58,853] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 222064                                              |
[2024-03-13 19:42:58,884] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 233665                                              |
[2024-03-13 19:42:58,889] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 242458                                              |
[2024-03-13 19:42:58,918] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 218375                                              |
[2024-03-13 19:42:58,925] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 208864                                              |
[2024-03-13 19:42:58,930] [builder.py:2230]           [WARNING]  | Convert Warning: No TVDb ID Found for TMDb ID: 242269                                              |
[2024-03-13 19:42:58,935] [overlays.py:440]           [DEBUG]    | Traceback (most recent call last):                                                                 |
                                                                 |   File "/modules/overlays.py", line 408, in compile_overlays                                       |
                                                                 |     builder.filter_and_save_items(builder.gather_ids(method, value))                               |
                                                                 |   File "/modules/builder.py", line 2228, in filter_and_save_items                                  |
                                                                 |     tvdb_id = self.config.Convert.tmdb_to_tvdb(input_id, fail=True)                                |
                                                                 |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                |
                                                                 |   File "/modules/convert.py", line 142, in tmdb_to_tvdb                                            |
                                                                 |     tvdb_id = self.config.TMDb.convert_from(tmdb_id, "tvdb_id", False)                             |
                                                                 |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                             |
                                                                 |   File "/modules/tmdb.py", line 192, in convert_from                                               |
                                                                 |     item = self.get_movie(tmdb_id) if is_movie else self.get_show(tmdb_id)                         |
                                                                 |                                                     ^^^^^^^^^^^^^^^^^^^^^^                         |
                                                                 |   File "/modules/tmdb.py", line 235, in get_show                                                   |
                                                                 |     return TMDbShow(self, tmdb_id, ignore_cache=ignore_cache)                                      |
                                                                 |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                      |
                                                                 |   File "/modules/tmdb.py", line 162, in __init__                                                   |
                                                                 |     self.seasons = [TMDbSeason(s) for s in loop]                                                   |
                                                                 |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                   |
                                                                 |   File "/modules/tmdb.py", line 162, in <listcomp>                                                 |
                                                                 |     self.seasons = [TMDbSeason(s) for s in loop]                                                   |
                                                                 |                     ^^^^^^^^^^^^^                                                                  |
                                                                 |   File "/modules/tmdb.py", line 83, in __init__                                                    |
                                                                 |     self.name = data.split(":")[1] if isinstance(data, str) else data.name                         |
                                                                 |                 ~~~~~~~~~~~~~~~^^^                                                                 |
                                                                 | IndexError: list index out of range                                                                |
                                                                 |                                                                                                    |
[2024-03-13 19:42:58,936] [overlays.py:441]           [ERROR]    | Unknown Error: list index out of range                                                             |
[2024-03-13 19:42:58,937] [overlays.py:442]           [INFO]     |                                                                                                    |

```

### Relevant Collection/Overlay/Playlist Definition

```yml
templates:
  # TEXT CENTER
  TV_Top_TextCenter:
    sync_mode: sync
    builder_level: show
    overlay:
      name: text(<<text>>)
      horizontal_offset: 0
      horizontal_align: center
      vertical_offset: 0
      vertical_align: top
      font: config/fonts/Juventus-Fans-Bold.ttf
      font_size: 70
      font_color: <<color>>
      group: TV_Top_TextCenter
      weight: <<weight>>
      back_color: <<back_color>>
      back_width: 1920
      back_height: 90

overlays:
# RETURNING 14/03/2024
  TV_Top_TextCenter_Returning_14/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 14/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/14/2024
      air_date.lte: 03/14/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 15/03/2024
  TV_Top_TextCenter_Returning_15/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 15/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/15/2024
      air_date.lte: 03/15/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 16/03/2024
  TV_Top_TextCenter_Returning_16/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 16/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/16/2024
      air_date.lte: 03/16/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 17/03/2024
  TV_Top_TextCenter_Returning_17/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 17/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/17/2024
      air_date.lte: 03/17/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 18/03/2024
  TV_Top_TextCenter_Returning_18/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 18/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/18/2024
      air_date.lte: 03/18/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 19/03/2024
  TV_Top_TextCenter_Returning_19/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 19/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/19/2024
      air_date.lte: 03/19/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 20/03/2024
  TV_Top_TextCenter_Returning_20/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 20/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/20/2024
      air_date.lte: 03/20/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 21/03/2024
  TV_Top_TextCenter_Returning_21/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 21/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/21/2024
      air_date.lte: 03/21/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 22/03/2024
  TV_Top_TextCenter_Returning_22/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 22/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/22/2024
      air_date.lte: 03/22/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 23/03/2024
  TV_Top_TextCenter_Returning_23/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 23/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/23/2024
      air_date.lte: 03/23/2024
      with_status: 0
      limit: 1000
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 24/03/2024
  TV_Top_TextCenter_Returning_24/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 24/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/24/2024
      air_date.lte: 03/24/2024
      with_status: 0
      limit: 1000
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 25/03/2024
  TV_Top_TextCenter_Returning_25/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 25/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/25/2024
      air_date.lte: 03/25/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 26/03/2024
  TV_Top_TextCenter_Returning_26/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 26/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/26/2024
      air_date.lte: 03/26/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 27/03/2024
  TV_Top_TextCenter_Returning_27/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 27/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/27/2024
      air_date.lte: 03/27/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 28/03/2024
  TV_Top_TextCenter_Returning_28/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 28/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/28/2024
      air_date.lte: 03/28/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 29/03/2024
  TV_Top_TextCenter_Returning_29/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 29/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/29/2024
      air_date.lte: 03/29/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 30/03/2024
  TV_Top_TextCenter_Returning_30/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 30/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/30/2024
      air_date.lte: 03/30/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 31/03/2024
  TV_Top_TextCenter_Returning_31/03/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 31/03"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 03/31/2024
      air_date.lte: 03/31/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 01/04/2024
  TV_Top_TextCenter_Returning_01/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 01/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/01/2024
      air_date.lte: 04/01/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 02/04/2024
  TV_Top_TextCenter_Returning_02/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 02/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/02/2024
      air_date.lte: 04/02/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 03/04/2024
  TV_Top_TextCenter_Returning_03/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 03/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/03/2024
      air_date.lte: 04/03/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 04/04/2024
  TV_Top_TextCenter_Returning_04/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 04/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/04/2024
      air_date.lte: 04/04/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 05/04/2024
  TV_Top_TextCenter_Returning_05/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 05/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/05/2024
      air_date.lte: 04/05/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 06/04/2024
  TV_Top_TextCenter_Returning_06/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 06/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/06/2024
      air_date.lte: 04/06/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 07/04/2024
  TV_Top_TextCenter_Returning_07/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 07/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/07/2024
      air_date.lte: 04/07/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 08/04/2024
  TV_Top_TextCenter_Returning_08/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 08/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/08/2024
      air_date.lte: 04/08/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 09/04/2024
  TV_Top_TextCenter_Returning_09/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 09/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/09/2024
      air_date.lte: 04/09/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 10/04/2024
  TV_Top_TextCenter_Returning_10/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 10/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/10/2024
      air_date.lte: 04/10/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 11/04/2024
  TV_Top_TextCenter_Returning_11/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 11/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/11/2024
      air_date.lte: 04/11/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 12/04/2024
  TV_Top_TextCenter_Returning_12/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 12/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/12/2024
      air_date.lte: 04/12/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 13/04/2024
  TV_Top_TextCenter_Returning_13/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 13/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/13/2024
      air_date.lte: 04/13/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 14/04/2024
  TV_Top_TextCenter_Returning_14/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 14/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/14/2024
      air_date.lte: 04/14/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 15/04/2024
  TV_Top_TextCenter_Returning_15/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 15/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/15/2024
      air_date.lte: 04/15/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 16/04/2024
  TV_Top_TextCenter_Returning_16/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 16/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/16/2024
      air_date.lte: 04/16/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 17/04/2024
  TV_Top_TextCenter_Returning_17/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 17/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/17/2024
      air_date.lte: 04/17/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 18/04/2024
  TV_Top_TextCenter_Returning_18/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 18/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/18/2024
      air_date.lte: 04/18/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 19/04/2024
  TV_Top_TextCenter_Returning_19/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 19/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/19/2024
      air_date.lte: 04/19/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 20/04/2024
  TV_Top_TextCenter_Returning_20/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 20/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/20/2024
      air_date.lte: 04/20/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 21/04/2024
  TV_Top_TextCenter_Returning_21/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 21/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/21/2024
      air_date.lte: 04/21/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 22/04/2024
  TV_Top_TextCenter_Returning_22/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 22/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/22/2024
      air_date.lte: 04/22/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 23/04/2024
  TV_Top_TextCenter_Returning_23/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 23/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/23/2024
      air_date.lte: 04/23/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 24/04/2024
  TV_Top_TextCenter_Returning_24/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 24/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/24/2024
      air_date.lte: 04/24/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 25/04/2024
  TV_Top_TextCenter_Returning_25/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 25/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/25/2024
      air_date.lte: 04/25/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024

# RETURNING 26/04/2024
  TV_Top_TextCenter_Returning_26/04/2024:
    template:
    - name: TV_Top_TextCenter
      weight: 35
      text: "RETURNING 26/04"
      color: "#000000"
      back_color: "#e6aa24"
      horizontal_align: center
      vertical_align: top
      horizontal_offset: 0
      vertical_offset: 0
    tmdb_discover:
      air_date.gte: 04/26/2024
      air_date.lte: 04/26/2024
      with_status: 0
      limit: 500
    filters:
      last_episode_aired.before: 01/28/2024
```


### Logs

_No response_

--- a/CHANGELOG
+++ b/CHANGELOG
@@ -41,3 +41,5 @@ Fixed episode rating mass update operations
 Fixed metadata backup issue where Artist, Album, and Track ratings were not being backed up
 
 Various other Minor Fixes
+
+Closed legacy issues on GitHub

